internal/fourslash/tests/gen/completionsJSDocNoCrash3_test.go

```diff
@@ -11,7 +11,7 @@ import (
 
 func TestCompletionsJSDocNoCrash3(t *testing.T) {
 	t.Parallel()
-
+	t.Skip()
```

Comment:
do we have an easy way to eyeball these `Skip` tests for the future? Do we just assume every `Skip` in fourslash is a bug?

Reply:
Some are bugs, some are missing features, e.g. JSDoc tests. The way to find out why they fail is to re-generate the tests without `Skip` and look at the error messages.